### ISSUE TITLE
Top-1 accuracy on the CIFAR100 dataset

### ISSUE THREAD

**Issue by HIT-lcccc** [pengzhiliang/Conformer#24]
Happy Chinese New Year. I tried to use the command "python main.py --model Conformer_tiny_patch16 --data-set CIFAR --batch-size 512 --lr 0.001 --num_workers 4 --data-path ../data --output_dir ./output/Conformer_tiny_patch16_batch512_lr1e3_epoch500_cifar100 --epochs 500 --input-size 32" to train Conformer on the CIFAR100 dataset with one tesla v100 GPU . But only got 45.89% accuracy. 
Would you mind providing the training parameters and top-1 accuracy on CIFAR 100 dataset?
![the validation accuracy in last epoch](https://user-images.githubusercontent.com/56480283/153692612-e95c0c46-2f58-4bc5-928a-426b07160bba.png)
